src/app/services/hero/hero.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+
+import { HeroModel } from 'models/hero/hero.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HeroService {
+  create(fly: boolean, life: number): HeroModel {
+    const heroBase = this.createBaseHero({ fly, life });
+    const newHero = heroBase;
+
+    return newHero;
+  }
+
+  createHurtedHero(fly: boolean, life: number): HeroModel {
```

Comment:
This method should receive single parameter as object withing fly AND name. Life should be applied automatically

Reply:
or 2 parameters, fly and name

Reply:
if it's single parameters why it has to be an object. For this particular case, they are only two. As you told me when they are two, you can pass them as single ones

Reply:
> or 2 parameters, fly and name

Look previous comment